### DISCUSSION TITLE
Issue with changes in template content - "No changes to save"

### DISCUSSION THREAD

**Discussion by cristinagenduso** [emailjs-com/help#95] [Q&A] [answered]
Hi, I need your help!

Whenever I modify the content of my template, I am unable to save it. The 'Edit Content' tab allows me to view the new content, but the content remains unchanged in the template overview. I've attempted to use various browsers, devices, and even incognito mode. Nothing works! I always get an alert saying 'No changes to save'. If I attempt to modify anything else other than the content, my template will be updated. I really don't know what to do. I would appreciate your help, thank you!

**Comment by xr0master** [comment]
Can you explain step by step your actions? Do you "Apply Changes" after the content editing?

**Reply by cristinagenduso** [reply]
Of course, I attach some screenshots.
![First](https://github.com/emailjs-com/help/assets/83962372/7372e6af-5acb-4728-b359-c08fb7e0d155)
![Second](https://github.com/emailjs-com/help/assets/83962372/a7275a11-a300-4435-b28a-78c9b792ec99)
As you can see from the first image, the content in the edit tab is actually updated. However, as soon as you close the template and save the template, no changes are detected and the preview shows that the content has not been changed.
![third](https://github.com/emailjs-com/help/assets/83962372/684cab65-2fdd-4690-94e0-1a1abfa7e77c)
The last image instead shows how changing, for example, the subject of the mail, it is actually updated and saved in the template.
However, the content never updates in any way.

**Reply by xr0master** [reply]
> as soon as you close the template

How do you do it?

**Reply by cristinagenduso** [reply]
I made a quick video to show you. Here's the link. Am I doing something wrong? As mentioned before, I tried using the incognito mode too, but nothing changed.
https://we.tl/t-0kKFKgooHu

**Reply by xr0master** [reply, marked as answer]
You are pressing on "x" button, and it means "cancel changes and close" the editor.
